mllib/src/main/scala/org/apache/spark/ml/feature/MinMaxScaler.scala

```diff
@@ -232,7 +233,9 @@ object MinMaxScalerModel extends MLReadable[MinMaxScalerModel] {
     override def load(path: String): MinMaxScalerModel = {
       val metadata = DefaultParamsReader.loadMetadata(path, sc, className)
       val dataPath = new Path(path, "data").toString
-      val Row(originalMin: Vector, originalMax: Vector) = sparkSession.read.parquet(dataPath)
+      val data = sparkSession.read.parquet(dataPath)
+      val Row(originalMin: Vector, originalMax: Vector) = MLUtils.convertVectorColumnsToML(
+        data, "originalMin", "originalMax")
```

Comment:
nit:

~~~scala
val Row(originalMin: Vector, originalMax: Vector) =
  MLUtils.convertVectorColumnsToML(data, "originalMin", "originalMax"))
    .select("originalMin", "originalMax")
~~~

It is slightly hard to read with `data` and `.select` having the same indentation.